libs/wingsdk/src/target-tf-azure/function.ts

```diff
@@ -155,6 +162,9 @@ export class Function extends cloud.Function {
         applicationStack: {
           nodeVersion: functionNodeVersion,
         },
+        applicationInsightsConnectionString:
+          this.applicationInsights.connectionString,
+        applicationInsightsKey: this.applicationInsights.instrumentationKey,
```

Comment:
(just for my own curiosity) What does this do?

Reply:
This enables logs and debugging to the function on Azure

Reply:
Got it - thanks! (Cloud providers always come up with such intuitive names 🙃)